source/draw/Mesh.ooc

```diff
@@ -6,9 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-version(!gpuOff) {
```

Comment:
Yes, we want to remove as much as possible of the `Gpu` abstraction layer anyway.